kotlin/src/main/kotlin/com/example/android/learn-kotlin/java/com/example/kotlin_samples/KotlinSamples.kt

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.android.kotlin.snippets
```

Comment:
could we use a different name for the folder?  snippets name is a little redundant when they're all snippets and the repo is called snippets. 

Also the package name is not matching the folder structure, the folder structure looks like it still has an underscore in it